src/main/resources/manifest.json

```diff
@@ -1,14 +1,14 @@
 [
   {
     "packageName": "fhir.r4.ukcore.stu1",
-    "version": "0.5.1"
+    "version": "1.0.0-pre-release"
   },
   {
     "packageName": "uk.nhsdigital.r4.test",
-    "version": "2.6.16-prerelease"
```

Comment:
Packages have different naming conventions; difference between **pre-release** on line 4 **prerelease** on line 8 is intentional